.bunnyshell/templates/express-react-rdspostgres/bunnyshell.yaml

```diff
@@ -0,0 +1,107 @@
+kind: Environment
+name: Express / React / AWS RDS PostgreSQL boilerplate
+type: primary
+environmentVariables:
+    DB_NAME: bunny_books
+components:
+    -
+        kind: Application
+        name: api
+        gitRepo: 'https://github.com/bunnyshell/templates.git'
+        gitBranch: express-rest-api/v0.4.0
+        gitApplicationPath: components/express-rest-api
+        dockerCompose:
+            build:
+                context: ./components/express-rest-api
+                dockerfile: .docker/Dockerfile
+                target: dev
+            environment:
+                FRONTEND_URL: 'https://{{ components.app.ingress.hosts[0].hostname }}'
+                POSTGRES_DB: '{{ env.vars.DB_NAME }}'
+                POSTGRES_HOST: '{{ components.db-rds.exported.DB_HOST }}'
+                POSTGRES_PASSWORD: '{{ components.db-rds.exported.DB_PASSWORD }}'
+                POSTGRES_USER: '{{ components.db-rds.exported.DB_USERNAME }}'
```

Comment:
The keys need to be ordered, otherwise they will show up on the diff when something else changes in the template.

Reply:
done